src/frontend/e2e/pages/upload.ts

```diff
@@ -0,0 +1,112 @@
+import { Page } from "@playwright/test";
```

Comment:
I don't feel like I'm clear on when to create a class like this one for use in tests. Will you include that in the documentation?

Reply:
i'll make a follow up documentation ticket for this

Reply:
https://app.shortcut.com/genepi/story/215675/documentation-when-to-use-classes-for-test-page-objects